### ISSUE TITLE
Update boto to 2.42.0 to fix unicode issue

### ISSUE THREAD

**Issue by paurullan** [APSL/docker-thumbor#43]
The `requirements.txt` currently installs 2.39.0, which has a nasty utf8 bug that it is solved in 2.42.0.

**Comment by eduherraiz** [comment]
updated in the the new future relase 6.1.3. The tag is ready in the docker hub.

**Comment by paurullan** [comment]
It seems that finally the error was on botocore.

Temporary fix on:
https://github.com/APSL/botocore/tree/release_1.2.x

```
pip install git+https://github.com/APSL/botocore.git@b497766
```

**Comment by eduherraiz** [comment]
Fixed again in the 6.1.3 pre-release

**Comment by eduherraiz** [comment]
Fixed and merged to master in 6.1.5
